conda/environments/nv_ingest_environment.yml

```diff
@@ -47,7 +48,8 @@ dependencies:
   - pip
   - pip:
       - llama-index-embeddings-nvidia
-      - nvidia-riva-client
       - opencv-python # For some reason conda cant solve our req set with py-opencv so we need to use pip
       - pymilvus>=2.5.0
       - pymilvus[bulk_writer, model]
+      - ffmpeg>=1.1
```

Comment:
The package nv-ingest uses is actually `ffmpeg-python` which is already listed on line 13.